runtime/queries/nasm/highlights.scm

```diff
@@ -0,0 +1,129 @@
+(comment) @comment
+
+(label) @label
+
+[
+  (preproc_expression)
+  (line_here_token)
+  (section_here_token)
+] @variable.builtin
+(preproc_expression
+  "]" @variable.builtin)
+(preproc_expression
+  "}" @variable.builtin)
+
+(unary_expression
+  operator: _ @operator.unary)
+(binary_expression
+  operator: _ @operator.binary)
+(conditional_expression
+  "?" @operator
+  ":" @operator)
+
+[
+  ":"
+  ","
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+(instruction_prefix) @keyword
+(actual_instruction
+  instruction: (word) @function)
+
+(call_syntax_expression
+  base: (word) @function)
+
+(size_hint) @type
+(struc_declaration
+  name: (word) @type)
+(struc_instance
+  name: (word) @type)
+
+(effective_address
+ hint: _ @type)
+(effective_address
+ segment: _ @constant.builtin)
+
+(register) @constant.builtin
+
+(number_literal) @number.integer
+(string_literal) @string
+(float_literal) @number.float
+(packed_bcd_literal) @number.integer
+
+((word) @constant
+  (#match? @constant "^[A-Z_][?A-Z_0-9]+$"))
+((word) @constant.builtin
+  (#match? @constant.builtin "^__\\?[A-Z_a-z0-9]+\\?__$"))
+(word) @variable
+
+(preproc_arg) @keyword
```

Comment:
If the preprocessor stuff is like the C preprocessor, these might fall under `@keyword.directive` https://docs.helix-editor.com/master/themes.html#syntax-highlighting

Reply:
I think you're right that `@keyword.directive` makes more sense. I've switched it to that in df213757f80e48498dfb5ea48de678c296af0f69 along with a few other improvements.